chatter.js

```diff
@@ -1,12 +1,80 @@
 Chatter = {
-    options: {
-        messageLimit: 30,
-        nickProperty: "username",
+  options: {
+    messageLimit: 30,
+    nickProperty: "username",
+  }
+};
+
+Chatter.configure = function (opts) {
+  _.extend(this.options, opts);
+};
+
+/**
+ * @summary Adds user to Chatter and defines its privileges
+ * @locus Server
+ * @param {string} userId Unique string identifying the users of your application.
+ * @param {string} userType Defines the restricitons on the new user. Can either be set to "admin" or "standard".
+ * @returns {string} userId
+ */
+Chatter.addUser = function(userId, userType) {
+   check(userId, String);
+   check(userType, Match.Maybe(String));
+  const user = Meteor.users.findOne({_id: userId});
+
+  Chatter.User.upsert({
+    userId: userId
+  }, {
+    $set: {
+      userType: userType,
+      nickname: getNickname(user)
     }
+  });
+  return Chatter.User.findOne({userId: userId})._id;
 };
 
+/**
+ * @summary Adds empty room to Chatter.
+ * @locus Server
+ * @param {string} rommName
+ * @returns {string} roomId
+ */
+Chatter.addRoom = function(roomName) {
```

Comment:
Sure you want to create such an empty room without knowing who the owner is, when it was made etc? If you are using astronomy with the 'timestamp' behaviour, you get createdAt propery for free when using the proper way of creating new Astronomy objects, `new Chatter.Room({ ... }).save()`